### PR TITLE
Minor docs cleanup

### PR DESCRIPTION
**Pull request by navidcy**
I noticed that there were a few docstrings that had an empty line between them and the method they were documenting. This empty line detaches the docstring from the method so Documented does not include it in the Library (Documenter does not know that those docstrings belong to those methods).

I removed the disclaimer from the Docs/Appendix/Library page. @jagoosw is there still methods/types whose docs don't appear in the library page? Note that since we have `private = false`, only exported methods/types should appear here.

### CODE REVIEW

**Comment by navidcy** [comment]
I also removed the tilmestep wizard from the first example. I thought it's better to keep it as simple as possible for the first example one sees. The particular buoyancy front example keeps CFL < 0.3 with Δt =  2sec.

File: docs/src/index.md [hunk 1]
```diff
@@ -100,15 +95,15 @@ nothing #hide
...
 ![buoyancy_front](buoyancy_front.mp4)
 
-In this example `OceanBioME` is providing the `biogeochemistry` and the remainder is taken care of by `Oceananigans`. For comprehensive documentation of the physics modelling see [Oceananigans' Documentation](https://clima.github.io/OceananigansDocumentation/stable/); for biogeochemistry and other features we provide read below.
+In the example above, `OceanBioME.jl` provides the `biogeochemistry` and everything else is taken care of by `Oceananigans.jl`. For comprehensive documentation of the physics modelling see [Oceananigans' Documentation](https://clima.github.io/OceananigansDocumentation/stable/); for biogeochemistry and other features we provide read below.
 
 ## Places to find OceanBioME information
 
 * This documentation, which provides
-    * example scripts,
+    * documented examples (e.g., start from the [single-column model](@ref OneD_column)),
```
Comment (navidcy) [inline docs/src/index.md:103]:
```suggestion
    * documented examples (browse them starting, e.g., from the [single-column model](@ref OneD_column)),
```

**Comment by navidcy** [comment]
@jagoosw I don't know if I messed up with the edits in [eea66e9](https://github.com/OceanBioME/OceanBioME.jl/pull/114/commits/eea66e970531ae6294733729745087f5c45ba47e). Have a look.

File: docs/src/index.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
...
 
-OceanBioME.jl is supported through grants from the [Center for Climate Repair at Cambridge](https://www.climaterepair.cam.ac.uk/) and the [Gordon and Betty Moore Foundation](https://www.moore.org/). 
+OceanBioME.jl currently provides a core of several biogeochemical models (Nutrient--Phytoplankton--Zooplankton--Detritus (NPZD) and [LOBSTER](https://doi.org/10.1029/2004JC002588), a medium complexity model, and air-sea gas exchange models to provide appropriate top boundary conditions. [PISCES](https://doi.org/10.5194/gmd-8-2465-2015) and also sediment models for the benthic boundary are in our future development plans.
```
Comment (jagoosw) [inline docs/src/index.md:5]:
```suggestion
OceanBioME.jl currently provides a core of several biogeochemical models (Nutrient--Phytoplankton--Zooplankton--Detritus (NPZD) and [LOBSTER](https://doi.org/10.1029/2004JC002588), a medium complexity model, air-sea gas exchange models to provide appropriate top boundary conditions, and sediment models to for the benthic boundary. [PISCES](https://doi.org/10.5194/gmd-8-2465-2015) and other higher complexity models are in our future development plans.
```
We do have a sediment model now although it hasn't been tested very much. I think its still appropriate to mention it here though?

File: src/Boundaries/Boundaries.jl [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
 Boundary conditions for air/sea and sediment flux. 
 
-Currently implimented:
+Currently implemented:
+
 - gasexchange ([Wanninkhof1992](@cite))
-    - Generic air sea flux  model describted by Wanninkhof, 1992 but only setup for CO₂ and O₂
-    - Forces the DIC and ocygen fields, and requires temp (in centigrade) and salinity, plus current DIC and ALK concentration
+  - Generic air-sea flux model described by Wanninkhof, 1992 but only setup for CO₂ and O₂.
+  - Forces the DIC and ocygen fields, and requires temp (in centigrade) and salinity, plus
```
Comment (jagoosw) [inline src/Boundaries/Boundaries.jl:8]:
```suggestion
  - Forces the DIC and oxygen fields, and requires temp (in centigrade) and salinity, plus
```

**Review by jagoosw — APPROVED** [review]
This looks good to me, thank you!

**Comment by navidcy** [comment]
cool, feel free to merge :)

**Comment by jagoosw** [comment]
I just thought, given that the Eady problem example has random initial conditions, it might stop it from intermittently failing if we set a random seed.

**Pull request merged**